docs/faq.md

```diff
@@ -49,6 +49,16 @@ You can use the `/forcesell all` command from Telegram.
 
 Please look at the [advanced setup documentation Page](advanced-setup.md#running-multiple-instances-of-freqtrade).
 
+### I'm getting "Missing data fillup" messages in the log
+
+This message is just a warning that the latest candles had missing candles in them.
+Depending on the exchange, this can indicate that the pair didn't have a trade for `<yourtimeframe>` - and the exchange does only return candles with volume.
```

Comment:
```suggestion
Depending on the exchange, this can indicate that the pair didn't have a trade for the timeframe you are using - and the exchange does only return candles with volume.
```